device-types/Cisco/ESA-C390.yaml

```diff
@@ -0,0 +1,30 @@
+---
+manufacturer: Cisco
+model: ESA C390
+slug: cisco-esa-c390
+part_number: ESA-C390-K9
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Console
+    type: rj-45
```

Comment:
You are missing the RPC port.

Not sure if it is best to classify it as a Console or an interface though.

Reply:
I've added the PRC Port to then Console Section. I think this fits best in this section.